parm/config/config.base.emc.dyn

```diff
@@ -180,15 +233,15 @@ export gfs_cyc=@gfs_cyc@ # 0: no GFS cycle, 1: 00Z only, 2: 00Z and 12Z only, 4:
 # GFS output and frequency
 export FHMIN_GFS=0
 
-export FHMAX_GFS_00=384
-export FHMAX_GFS_06=384
-export FHMAX_GFS_12=384
-export FHMAX_GFS_18=384
+export FHMAX_GFS_00=${FHMAX_GFS_00:-384}
+export FHMAX_GFS_06=${FHMAX_GFS_06:-384}
+export FHMAX_GFS_12=${FHMAX_GFS_12:-384}
+export FHMAX_GFS_18=${FHMAX_GFS_18:-384}
 export FHMAX_GFS=$(eval echo \${FHMAX_GFS_$cyc})
 
-export FHOUT_GFS=3
-export FHMAX_HF_GFS=0
-export FHOUT_HF_GFS=1
+export FHOUT_GFS=${FHOUT_GFS:-3}
+export FHMAX_HF_GFS=${FHMAX_HF_GFS:-0}
+export FHOUT_HF_GFS=${FHOUT_HF_GFS:-1}
```

Comment:
I think the current develop of the UFS now no longer has `HF` as an input.
I think they want all forecast hours explicitly defined as a list.
Please check though.